.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,10 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
don't revision this file.

Reply:
why don't add /.idea in.gitignore file then ?